ui/pages/AnvilWorkspace/components/LoadWorkspaceDataForm.jsx

```diff
@@ -104,13 +104,23 @@ const createProjectFromWorkspace = postWorkspaceValues(
   window.location.href = `/project/${responseJson.projectGuid}/project_page`
 })
 
-const FORM_WIZARD_PAGES = [
-  { fields: [AGREE_CHECKBOX], onPageSubmit: postWorkspaceValues('grant_access') },
-  { fields: [DATA_BUCK_FIELD, SAMPLE_TYPE_FIELD, REQUIRED_GENOME_FIELD], onPageSubmit: postWorkspaceValues('validate_vcf') },
-  { fields: [PROJECT_DESC_FIELD, UPLOAD_PEDIGREE_FIELD] },
-]
-
-const LoadWorkspaceDataForm = React.memo(({ params }) => (
+const addDataFromWorkspace = projectGuid => values => (
+  new HttpRequestHelper(`/api/project/${projectGuid}/add_workspace_data`, (responseJson) => {
+    if (responseJson.success) {
+      window.location.href = `/project/${projectGuid}/project_page`
```

Comment:
The Wizard is created in a Modal. I use a redirect to jump to the project page after submission. It may be more user-friendly by just closing the Modal.  But it needs some changes to the Wizard component. Should I make the change?

Reply:
So I didn't actually think this through on the ticket, so I'm sorry for not being more clear upfront. Thinking about this now what I think we actually want is after submission succeeds to keep the modal open and show a message telling them that their request was submitted and they should get their data in about a week, so they don't get confused about why there is no data. The exact message I would use is "Your request to load data has been submitted. Loading data from AnVIL to seqr is a slow process, and generally takes a week. You will receive an email letting you know once your new data is available."

I think the best way to do this is to modify the `FormWizard` component to take an optional `successMessage` prop and if that prop exists to show that message instead of the form after submission succeeds. I would add a field to the state like `formSubmitSuceeded` and then use the `onSubmitSucceeded` prop for the final page to set that state field